app/routes/missions.svom/route.mdx

```diff
@@ -0,0 +1,67 @@
+
+The [Space-based multi-band astronomical Variable Objects Monitor (SVOM)](https://www.svom.eu/en/the-svom-mission/) is a French-Chinese mission, result of a collaboration between the two national space agencies, [China National Space Administration (CNSA)](https://www.cnsa.gov.cn/english/index.html) and [Centre national d'études spatiales (CNES)](https://irfu.cea.fr/dap/en/Phocea/Vie_des_labos/Ast/ast_technique.php?id_ast=2276). SVOM mission is dedicated to the study of the most distant explosions of stars, the gamma-ray bursts (GRBs).
+
+The SVOM spacecraft carries 4 main instruments of which 2 are French (ECLAIRs and MXT) and 2 are Chinese (GRM and VT). Transient alert messages will be sent for:
+
+ <div className="overflow-table">
+
+| Instruments                        | Energy Range            | Field of View | Localization                                   |
+| ---------------------------------- | ----------------------- | ------------- | ---------------------------------------------- |
+| ECLAIRs                            | 4 - 250 keV             | 2 ster        | ~8' radius (statistical, 90% confidence)       |
+| Gamma Ray burst Monitor (GRM)      | 15 - 5000 keV           | 2.6 ster      | ~1-10° radius (statistical, 90% confidence)    |
+| Microchannel X-ray Telescope (MXT) | 15 - 5000 keV           | 1.1° x 1.1°   | ~10-100'' radius (statistical, 90% confidence) |
+| Visual Telescope (VT)              | Blue Band: 450 - 650 nm | 26' x 26'     | ~TBD'' radius (statistical, 90% confidence)    |
+|                                    | Red Band: 650 - 1000 nm |               |                                                |
+
+ </div>
+
+#### JSON-Serialized GCN Notice Types in GCN Kafka
+
+SVOM distributes alerts and localizations detections of gamma-ray transients. These notices are published on the GCN Kafka and VOEvent topics as `gcn.notices.svom` and `gcn.notices.svom.voevent`, respectively.
+
+ <div className="overflow-table">
+
+| Type               | Contents                                                                                  | Latency   |
+| ------------------ | ----------------------------------------------------------------------------------------- | --------- |
+| `trigger`          | eclairs.trigger.wakeup, eclairs.trigger.not-slewing, eclairs.trigger.slewing, grm.trigger | ~1-10 sec |
+| `refined`          | eclairs.refined, grm.refined, eclgrm.refined                                              | ~10 min   |
+| `mxt.localisation` | mxt.localisation                                                                          | ~1 min    |
+
+ </div>
+
+#### Yearly Trigger Rates
+
+ <div className="overflow-table">
+
+| Instrument        | Type                             | Yearly Rates |
+| ----------------- | -------------------------------- | ------------ |
+| ECLAIRs           | GRBs (TBD)                       | TBD          |
+| Gamma Ray Monitor | GRBs, and other transients (TBD) | TBD          |
```

Comment:
(TBD) not needed if it says "and other transients)

Use either full names or acronyms, not a combination, so "Gamma ray Monitor" should be "GRM"